### ISSUE TITLE
(Feature request) How do I add a text line when the code block is the last line?

### ISSUE THREAD

**Issue by Joackk** [zadam/trilium#3955]
### Describe feature

Sometimes I habitually make the code block as the last line, but then I need to enter new text, but I can't add a blank line after the code block, a direct enter will only add a line of code block, I can only delete a few lines of the code block and then recreate the block, can I add a blank line for it directly like a table?

**Comment by Shultc** [comment]
This is so counterintuitive, that I needed to find this issue, to find the solution.
Since exiting other things, like bullet list for example, you need only two enters, I don't think this solution is acceptable.

Please reopen.

**Comment by zadam** [comment]
> like bullet list for example, you need only two enters

Because in a bullet list you typically don't have empty lines. In code block you do.

> Please reopen.

This is a behavior of CKEditor and I want to avoid changing the behavior of it as much as possible.

**Comment by Ranglage** [comment]
> > This is a behavior of CKEditor and I want to avoid changing the behavior of it as much as possible.
> 
> I totally understand that. Nevertheless, if you need to google how to exit something, we are approaching vim-meme territory.
> 
> Maybe there is possibility to add some "tip"? Or always automatically add new line after code block, when it was inserted?

I totally agree with you. I have to google (and find nothing) and find within GitHub issues to get this answer.
